easybuild/easyconfigs/b/Bracken/Bracken-2.6.0-GCCcore-9.3.0.eb

```
@@ -0,0 +1,56 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'Bracken'
+version = '2.6.0'
+
+homepage = 'https://ccb.jhu.edu/software/bracken/'
+description = """Bracken (Bayesian Reestimation of Abundance with KrakEN) 
+is a highly accurate statistical method that computes the abundance of 
+species in DNA sequences from a metagenomics sample. Braken uses the 
+taxonomy labels assigned by Kraken, a highly accurate metagenomics 
+classification algorithm, to estimate the number of reads originating 
+from each species present in a sample. Kraken classifies reads to the 
+best matching location in the taxonomic tree, but does not estimate 
+abundances of species. We use the Kraken database itself to derive 
+probabilities that describe how much sequence from each genome is 
+identical to other genomes in the database, and combine this information 
+with the assignments for a particular sample to estimate abundance at 
+the species level, the genus level, or above. Combined with the Kraken 
+classifier, Bracken produces accurate species- and genus-level abundance 
+estimates even when a sample contains two or more near-identical species.
+
+NOTE: Bracken is compatible with both Kraken 1 and Kraken 2. However, the 
+default kmer length is different depending on the version of Kraken used. 
+If you use Kraken 1 defaults, specify 31 as the kmer length. If you use 
+Kraken 2 defaults, specify 35 as the kmer length."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://github.com/jenniferlu717/Bracken/archive/']
+sources = ['v%(version)s.tar.gz']
+
+checksums = [
+    'fb1837d6f32b8f8c87353b9dc8a23d3418c348b00824a7064eb58f9bab11ea68',  # v%(version)s.tar.gz
+]
+
+builddependencies = [('binutils', '2.34')]
+
+# no need to build in parallel 
+parallel = 1
+
+build_cmd = "cd %(builddir)s/%(name)s-%(version)s/src && make"
```

@sassy-crick no need to change the build_cmd here, just update start_dir:
```suggestion
start_dir = 'src'
```

---

Thanks for pointing this out, that was something I did not know (or missed). I have updated the PR.